tailnet/coordinator.go

```diff
@@ -321,24 +255,27 @@ func (c *core) agentCount() int {
 
 // ServeClient accepts a WebSocket connection that wants to connect to an agent
 // with the specified ID.
-func (c *coordinator) ServeClient(conn net.Conn, id uuid.UUID, agent uuid.UUID) error {
+func (c *coordinator) ServeClient(conn net.Conn, id, agentID uuid.UUID) error {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
-	logger := c.core.clientLogger(id, agent)
+	logger := c.core.clientLogger(id, agentID)
 	logger.Debug(ctx, "coordinating client")
-	tc, err := c.core.initAndTrackClient(ctx, cancel, conn, id, agent)
+
+	ma := c.ServeMultiAgent(id)
```

Comment:
I think it's better to continue to use a `TrackedConn` for remote clients - the power of the `Enqueuable` abstraction is that the `core` can treat it equivalently to `MultiAgent`.

Then we won't need `SendUpdatesToConn()` as a separate function that essentially repeats most of `TrackedConn.SendUpdates()`.